config/channel/resources/kafkachannel-crd.yaml

```diff
@@ -144,8 +144,6 @@ spec:
             status:
               description: Status represents the current state of the KafkaChannel. This data may be out of date.
               type: object
-              required:
-                - address
               properties:
                 address:
```

Comment:
I miss something here....

this change was already done to 0.25 -> https://github.com/openshift-knative/eventing-kafka/pull/344/files


Related, we also regenerated our CI manifests -> https://github.com/openshift-knative/eventing-kafka/pull/384

But `openshift` folder is missing here`